resources/qml/UserSettingsPage.qml

```diff
@@ -0,0 +1,218 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+// SPDX-FileCopyrightText: 2022 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import "ui"
+import Qt.labs.platform 1.1 as Platform
+import QtQuick 2.15
+import QtQuick.Controls 2.3
```

Comment:
I don't think there's any reason to use old import versions since we require 5.15+ anyway.  Might as well just require the latest, and then we can remove them once the qt 6 port lands.

Reply:
We should probably update it everywhere, I think I just copied that from another file.